docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -3,104 +3,93 @@ title: Run load-balanced VMs on Azure for scalability and availability
 description: >-
   How to run multiple Linux VMs on Azure for scalability and availability.
 
-author: MikeWasson
+author: telmosampaio
 
-ms.date: 11/22/2016
+ms.date: 09/07/2017
 
 pnp.series.title: Linux VM workloads
 pnp.series.next: n-tier
 pnp.series.prev: single-vm
 ---
+
 # Run load-balanced VMs for scalability and availability
 
-This reference architecture shows a set of proven practices for running several Linux virtual machines (VMs) behind a load balancer, to improve availability and scalability. This architecture can be used for any stateless workload, such as a web server, and is a building block for deploying N-tier applications. [**Deploy this solution**.](#deploy-the-solution) 
+This reference architecture shows a set of proven practices for running several Linux virtual machines (VMs) in a scaleset behind a load balancer, to improve availability and scalability. This architecture can be used for any stateless workload, such as a web server, and is a building block for deploying n-tier applications. [**Deploy this solution**.](#deploy-the-solution)
 
 ![[0]][0]
 
 ## Architecture
 
 This architecture builds on the one shown in [Run a Linux VM on Azure][single vm]. The recommendations there also apply to this architecture.
 
-In this architecture, a workload is distributed across several VM instances. There is a single public IP address, and Internet traffic is distributed to the VMs using a load balancer. This architecture can be used for a single-tier application, such as a stateless web application or storage cluster. It's also a building block for N-tier applications. 
+In this architecture, a workload is distributed across several VM instances. There is a single public IP address, and Internet traffic is distributed to the VMs using a load balancer. This architecture can be used for a single-tier application, such as a stateless web application.
 
-The architecture has the following components: 
+The architecture has the following components:
 
-* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set. 
+* **Resource group.** A [*resource group*][resource-manager-overview] is a container that holds related resources. You usually create resource groups for different resources in a solution based on their lifetime, and who will manage the resources.
```

Comment:
The second sentence has the comma splice I mentioned yesterday - may I suggest:

Resource groups are typically used to group resources so they can be managed by lifetime, owner, and other criteria.